extensions/2.0/Khronos/KHR_gaussian_splatting/README.md

```diff
@@ -141,9 +151,28 @@ For example, a new extension `EXT_gaussian_splatting_kernel_customShape` could b
 }]
 ```
 
+### Color Space
+
+The `colorSpace` property is an optional property that specifies the color space of the 3D Gaussian Splat when spherical harmonics are being used for the lighting. The color space is typically determined by the training process for the splats. The default value is `BT.709` to align with the base glTF specification. This color space value only applies to the 3D Gaussian splatting data and does not affect any other color data in the glTF.
+
+Renderers may ignore this property and assume `BT.709` based on their implementation's capabilities. When a renderer does not support the specified color space, it may clamp to the `BT.709` color space, but this may cause quality issues for the 3D Gaussian splatting data.
+
+Additional values can be added over time by defining an extension that adds new color spaces.
+
+#### Available Color Spaces
+
+| Color Space | Description |
+| --- | --- |
+| BT.709 | BT.709 color space. (Default) |
+| BT.2020-itu | BT.2020-ITU color space. |
+| BT.2020-hlg | BT.2020 HLG (Hybrid Log-Gamma) color space. |
+| BT.2020-pq | BT.2020 PQ (Perceptual Quantizer) color space. |
+| Display-P3 | Display P3 color space. |
+| linear | Linear color space. |
```

Comment:
This entry does not fit here. If the idea is to merge gamuts and transfer functions into combined enum values, there should be "BT.709-linear" and "BT.2020-linear" instead of this one.